picoquictest/tls_api_test.c

```diff
@@ -1338,17 +1341,17 @@ int tls_api_one_scenario_test(test_api_stream_desc_t* scenario,
 
 int tls_api_oneway_stream_test()
 {
-    return tls_api_one_scenario_test(test_scenario_oneway, sizeof(test_scenario_oneway), 0, 0, 0, 0, 70000, NULL);
+    return tls_api_one_scenario_test(test_scenario_oneway, sizeof(test_scenario_oneway), 0, 0, 0, 0, 100000, NULL);
```

Comment:
Why did you increase the max timeout?

Reply:
There is a regression in the duration of tests. I believe it is because PMTU discovery starts later in the new state machine. I am going to try fix that.

Reply:
Yes, that was the reason. Fixed now. Thanks for prodding me!

Reply:
Fixed.